app-containers/containers-image/containers-image-5.29.2-r1.ebuild

```diff
@@ -25,6 +25,12 @@ PATCHES=(
 	"${FILESDIR}"/moving-policy-json-default-yaml.patch
 )
 
+src_prepare() {
+	default
+	# following avoids downloading unnecessary modules
+	sed -i -e '/go list/d' Makefile || die
```

Comment:
This is a good case where a patch may be better, because you get told when it stops applying, as opposed to a sed which silently does nothing.

Reply:
done. Could you also take care of https://bugs.gentoo.org/921771